archetypes/shared-services/archetypeAzureDevopsPipeline.yml

```diff
@@ -8,9 +8,22 @@ variables:
 stages:
 - stage: Validate
   jobs:
+  - job: SetupValidationResourceGroup
+    pool: 
+      name: 'vdc-self-hosted'
+    steps:
+    - task: AzurePowerShell@4
+      displayName: "Setup Validation Resource Group"
+      inputs:
+        azureSubscription: 'vdc2-hub'
+        ScriptType: 'FilePath'
+        ScriptPath: 'orchestration/OrchestrationService/ValidationResourceGroupSetup.ps1'
+        ScriptArguments: '-ResourceGroupName vdc-validation-rg'
```

Comment:
what happens if a resoureGroupName is not passed?

Reply:
The script is called with "null" resulting in the script creating a resource group named null.

Reply:
This is a bug in the Azure Powershell v4 task. Even after removing the script arguments from the task, a null string is being passed as an argument to the script. Planning to add a Platform feedback for this task.